pigtail/build.gradle

```diff
@@ -1,21 +1,25 @@
 buildscript {
+
+        Properties constants = new Properties()
+    file("$projectDir/../constants.properties").withInputStream { constants.load(it) }
+
     ext {
-        corda_release_group = 'net.corda'
-        corda_release_version = '4.0'
-        corda_gradle_plugins_version = '4.0.42'
-        kotlin_version = '1.2.71'
-        junit_version = '4.12'
-        slf4j_version = '1.7.25'
-        quasar_version = '0.7.10'
-        netty_version = '4.1.22.Final' // Same version of Netty as Corda uses.
-        corda_platform_version = '4'
+        corda_release_group = constants.getProperty("cordaReleaseGroup")
+        corda_release_version = constants.getProperty("cordaVersion")
+        corda_gradle_plugins_version = constants.getProperty("gradlePluginsVersion")
+        kotlin_version = constants.getProperty("kotlinVersion")
+        junit_version = constants.getProperty("junitVersion")
+        quasar_version = constants.getProperty("quasarVersion")
+        log4j_version = constants.getProperty("log4jVersion")
+        slf4j_version = constants.getProperty("slf4jVersion")
+        corda_platform_version = constants.getProperty("platformVersion")
+        netty_version = constants.getProperty("nettyVersion")
     }
 
     repositories {
         mavenLocal()
         mavenCentral()
         jcenter()
-        maven { url 'https://ci-artifactory.corda.r3cev.com/artifactory/corda-releases' }
```

Comment:
I think you need to add this line back in for the pigtail sample? (Even though it's broken anyways)